intent-open-concepts/index.md

```diff
@@ -1,6 +1,17 @@
 ---
 title: Open Concept List
 ---
+<script>
+  function showmath (){
+      const ml =document.querySelectorAll("math");
+      for(const m of ml) {
+	  const md =  document.createElement("div");
```

Comment:
Where will this div element put. (I recently learned that it is often better to puth mathml elements in a span)

Reply:
@physikerwelt  so it's not putting the math element in a div it is removing the math element entirely.